src/user_management/user.resolver.js

```diff
@@ -1,9 +1,36 @@
+import { AuthenticationError } from 'apollo-server-express'
+
 import models from '~/src/service_providers/sequelize/models'
 
 export default {
+  Query: {
+    users: (_,args) => {
+      return models.user.findAll()
+    },
+    currentUser: {
+
+    },
+    logInUser: async (_,{email, password}) => {
+      const user = await models.user.findOne({
+        where: {
+          email
+        }
+      }) 
+      if (user && await user.validatePass(password)){
```

Comment:
Try using the new [Optional Chaining Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).
```suggestion
      if (await user?.validatePass(password)){
```